docs/src/pages/layout/grid/grid.md

```diff
@@ -23,8 +23,11 @@ The grid system is implemented with the `Grid` component:
 ## Spacing
 
 The responsive grid focuses on consistent spacing widths, rather than column width.
-Material design margins and columns follow an **8dp** square baseline grid.
-Spacing can be 8, 16, 24, 32 or 40dp wide.
+Material design margins and columns follow an **8px** square baseline grid.
+The spacing property is an integer in the [0, 8] interval.
```

Comment:
```suggestion
The spacing property is an integer between 0 and 8.
```

Reply:
However your notation is ambiguous about inclusive vs. exclusive intervals. If we can reword that to be explicit then I would agree that we should avoid mathematical notation.

Reply:
> However your notation is ambiguous about inclusive vs. exclusive intervals.

Fair point, although I imagine most people would read that as inclusive. Let's be explicit though:

"The spacing property is an integer between 0 and 8 inclusive."

or

"The spacing property is an integer in the range 0 to 8."

?

Reply:
I'm going with `The spacing property is an integer between 0 and 10 inclusive.`.